test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -17,9 +17,10 @@ Describe "TabCompletion" -Tags CI {
     }
 
     It 'Should complete abbreviated function' {
-        $res = (TabExpansion2 -inputScript 'pschrl' -cursorColumn 'pschr'.Length).CompletionMatches.CompletionText
+        function Test-AbbreviatedFunctionExpansion {}
```

Comment:
Why does this test change in this PR?

Reply:
Relying on PSReadLine in this test is unreliable as you may have multiple installs and it finds them.  Particularly with this change, on AzDevOps, it's starting PSCore6 which has PSRL, then starts PS7 which has PSRL which results in two instances with this change.  When AzDevOps switches to PS7, it'll show one again.  Better to have the test rely on something it controls.